### DISCUSSION TITLE
bug: button opens up profile different from mine

### DISCUSSION THREAD

**Discussion by giulianopz** [community/community#128802] [Mobile]
Hi,
starting from yesterday, when I click on the profile button on the bottom of the mobile app I see the profile of a user different from me. Please, see the pics below.

My real profile:
![Screenshot_20240617-040606.png](https://github.com/community/community/assets/45801466/17666598-a22b-47a4-93a2-3b86501f829f)

What I see after clicking on 'profile':
![Screenshot_20240617-040622.png](https://github.com/community/community/assets/45801466/bcefe8c4-4f52-4f4e-8427-16998daa53f9)

Logging in/out probably fixes the problem. I'm reporting the issue just for its security concerns (if any): luckily, it seems that I am just able to see the other user's profile but I can't modify/delete his repositories.

App version: GitHub Mobile for Android v1.165.0 (791)

**Comment by TheGiraffe3** [comment]
Try signing out on mobile and signing back in with the real user account.

**Reply by giulianopz** [reply]
Yep, I guess. I'm reporting the issue for its security concerns (if any). I added a statement to the issue with this regard. Thanks, anyway.

**Comment by Davdadev** [comment]
that the default icon 
, are you part of an organisation it could be that?
